client/document/index.jsx

```diff
@@ -111,6 +112,7 @@ class Document extends React.Component {
 					) }
 					{ badge && (
 						<div className="environment-badge">
+							{ preferencesHelper && <div className="environment is-prefs" /> }
```

Comment:
Just a side note that this "component" is likely to be refactored in the next time here https://github.com/Automattic/wp-calypso/pull/22625. I think there's nothing special you need to consider, just wanted to give you this information.